### PR TITLE
apply unified interface changes

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the precompiles pages with the new naming convention (camelCase instead of snake_case)

Goes with CN PR: https://github.com/PureStake/moonbeam-docs-cn/pull/156

### CODE REVIEW

File: .snippets/text/xcm/xcm-precompile-multilocation.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ```js
  struct Multilocation {
     uint8 parents;
-    bytes [] interior;
+    bytes[] interior;
```
Comment (albertov19) [inline .snippets/text/xcm/xcm-precompile-multilocation.md:4]: haha 🕵️‍♀️ - but I think it works both ways

**Review by albertov19 — APPROVED** [review]
Looks good. This something we can also change in the `CN` repo right away, right?

File: .snippets/text/xcm/xcm-precompile-multilocation.md (hunk 1 above)
Comment (eshaben) [reply .snippets/text/xcm/xcm-precompile-multilocation.md:4]: yeah just figured I would just be consistent with changes made in the moonbeam repo 🤷‍♀️

**Comment by eshaben** [comment]
> Looks good. This something we can also change in the `CN` repo right away, right?

yepp, linked to the CN repo PR in the description 🚀

**Pull request merged**